tests/test_functional.py

```diff
@@ -361,6 +361,19 @@ def test_save_update_note(self, user, testapp, rodne_cislo):
         assert "Lorem ipsum dolor sit amet,</textarea>" in res.text
         assert Note.query.count() == existing_notes + 1
 
+    @pytest.mark.parametrize("rodne_cislo", ["0010126521", "ahoj", 444])
```

Comment:
Jsi si jist, že rodné číslo _0010126521_ tam nemůže být? Jakože není v testovacích datech? Napadlo mě dát nějaké ne úplně neplatné, ale prakticky ano. Jakože s rokem 2050, nebo třeba třináctým měsícem, třicátým druhým dnem, nebo tak.

Reply:
Dobrá poznámka jistý si úplně jsem, bylo to náhodně zvolené číslo

Reply:
Pokud je jen náhodné, tak bych to raději prověřil.

Reply:
jj určitě mám v plánu na to ještě kouknout